### DISCUSSION TITLE
🌀 ES6 Support

### DISCUSSION THREAD

**Discussion by im-parsa** [DiscordlyCity/DiscordlyCity-SDK#4] [Q&A] [answered]
Is there anyway to convert Common JS to ES6 for modules to be able to use discords.city package ?

**Comment by EhsanFox** [comment, marked as answer]
We already have done that for ES6 Support, we use a package called **gen-esm-wrapper** which converts our CommonJS to ES6

we also have configured the package.json for ES6 Users.
